packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -331,6 +344,10 @@ export default defineComponent({
 
 			extensions.push(
 				...writableEditorExtensions,
+				keymap.of([
+					{ key: 'Ctrl-;', run: toggleComment },
```

Comment:
we already have `cmd /`, which a quick internal poll showed most of our engineers prefer. is there a reason you prefer this? also this would need to be added to `writableEditorExtensions`

Reply:
Hey, not time for that but sure whatever the shortcut if this does the job. I'm happily using this on our selfhosted version but we miss something similar on our cloud account so I guess it would have value for other customers :)

Reply:
`cmd /` should be working on cloud as well.

Reply:
not working on mac